bl2.c

```diff
@@ -20,15 +20,13 @@
 #define bh_rd(h, sz, off)						\
 	(le ## sz ## toh(*(uint ## sz ## _t *)((h) + off)))
 
-#define BL2IMG_TOTSZ	0xc000
-#define BL2RAND_SZ	0x10
-#define BL2HDR_SZ	0x40
-#define BL2HASH_SZ	0x200
+#define BL2HDR_SZ	0x1000
+#define BL2IV_SZ	0x10
+#define BL2BLKHDR_SZ	0x40
+#define BL2SIG_SZ	0x200
 #define BL2KEY_SZ	0xD80
 #define BL2KEYHDR_SZ	0x30
-#define BL2SHA2_LEN	0x20
-#define BL2BIN_SZ	(BL2IMG_TOTSZ -	(BL2RAND_SZ + BL2HDR_SZ +	\
-			 BL2HASH_SZ + BL2KEYHDR_SZ + BL2KEY_SZ))
+#define BL2SHA2_SZ	0x20
```

Comment:
Can't those macro name changes above be in previous patch ?

Reply:
Oh, s*it. I screwed up the commit when rebasing, sorry about that. Let me prepare a new PR.